numba_dppy/dppl_passbuilder.py

```diff
@@ -44,6 +45,9 @@ def default_numba_nopython_pipeline(state, pm):
         pm.add_pass(IRProcessing, "processing IR")
         pm.add_pass(WithLifting, "Handle with contexts")
 
+        # this pass rewrite name of functions
+        pm.add_pass(DPPLRewriteOverloadedFunctions, "dppl rewrite name of functions")
```

Comment:
`dppy`. Also, the pass description needs to be improved.

Reply:
Desctiption updated. Please check.